Xamarin.Forms.Xaml/MarkupExtensions/OnPlatformExtension.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Globalization;
+using System.Reflection;
+
+namespace Xamarin.Forms.Xaml
+{
+	[ContentProperty("Default")]
+	public class OnPlatformExtension : IMarkupExtension
+	{
+		public object Default { get; set; }
+		public object Android { get; set; }
+		public object GTK { get; set; }
+		public object iOS { get; set; }
+		public object macOS { get; set; }
+		public object Tizen { get; set; }
+		public object UWP { get; set; }
+		public object WPF { get; set; }
+
+		public IValueConverter Converter { get; set; }
+
+		public object ConverterParameter { get; set; }
+
+		public object ProvideValue(IServiceProvider serviceProvider)
+		{
+			var lineInfo = serviceProvider?.GetService<IXmlLineInfoProvider>()?.XmlLineInfo;
+			if (Android == null && GTK == null && iOS == null && 
+				macOS == null && Tizen == null && UWP == null && 
+				WPF == null && Default == null && string.IsNullOrEmpty(Other))
```

Comment:
doesn't build. `Other` is no longer defined

Reply:
🤦‍♂️